### PR TITLE
Items View - Minor UI Fixes

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
- Fixed removeTrailingZeros for formatFiat. It previously didn't affected returned value, formattedAmount.

- Item names are displayed in bold.

- Item icons are now smaller in size.

- Added BTC and SAT symbols, these will be displayed on catalog

### CODE REVIEW

File: lib/routes/charge/items/catalog_item.dart [hunk 1]
```diff
@@ -59,10 +60,32 @@ class CatalogItem extends StatelessWidget {
                     mainAxisAlignment: MainAxisAlignment.start,
                     mainAxisSize: MainAxisSize.min,
                     children: <Widget>[
-                      Text(
-                        _formattedPrice(),
-                        style: theme.transactionAmountStyle,
-                      )
+                      RichText(
```
Comment (roeierez) [inline lib/routes/charge/items/catalog_item.dart]: @erdemyerebasmaz  Can you explain this code that use substring?

File: lib/bloc/account/fiat_conversion.dart [hunk 2]
```diff
@@ -58,7 +58,7 @@ class FiatConversion {
     }
     if (removeTrailingZeros) {
       RegExp removeTrailingZeros = RegExp(r"([.]0*)(?!.*\d)");
-      formattedAmount.replaceAll(removeTrailingZeros, "");
+      formattedAmount = formattedAmount.replaceAll(removeTrailingZeros, "");
     }
```
Comment (erdemyerebasmaz) [inline lib/bloc/account/fiat_conversion.dart]: The changes were not being applied to returned value.

File: lib/bloc/user_profile/currency.dart [hunk 3]
```diff
@@ -6,35 +6,47 @@ import 'package:intl/number_symbols_data.dart';
 enum CurrencyID { BTC, SAT }
 
 class Currency extends Object {
-  final String symbol;
+  final String tickerSymbol;
   static const Currency BTC = Currency._internal("BTC");
```
Comment (erdemyerebasmaz) [inline lib/bloc/user_profile/currency.dart]: symbol is renamed to tickerSymbol as per image

File: lib/bloc/user_profile/currency.dart [hunk 4]
```diff
@@ -6,35 +6,47 @@ import 'package:intl/number_symbols_data.dart';
...
+
+  String get symbol {
+    switch (this.tickerSymbol) {
+      case "BTC":
+        return "₿";
+      case "SAT":
+        return "\$";
+      default:
+        return "₿";
+    }
```
Comment (erdemyerebasmaz) [inline lib/bloc/user_profile/currency.dart]: $ character is replaced with satoshi icon for _sat.otf_ font family

File: lib/routes/charge/items/catalog_item.dart [hunk 5]
```diff
@@ -83,12 +89,38 @@ class CatalogItem extends StatelessWidget {
...
-    return formatter.format(_itemInfo.price, includeCurencySuffix: true, removeTrailingZeros: true);
+  InlineSpan _buildCurrencySymbol() {
+    return _itemInfo.currency == "SAT"
+        ? WidgetSpan(
+            child: Container(
+              height: 20,
+              child: Align(
+                child: Text(_getSymbol(), // Icon
+                    style: TextStyle(fontSize: 13.4, fontFamily: 'SAT')),
+                alignment: Alignment.center,
```
Comment (erdemyerebasmaz) [inline lib/routes/charge/items/catalog_item.dart]: Custom fontFamily is used to display SAT icon.
Character for SAT icon has it's issues, such as it's alignment and font size is not consistent even with the font family(Ubuntu) it's derived from. 
The symbol is wrapped around with a center aligned container of height 20 to match style of price tags for other currencies.

File: lib/routes/charge/items/catalog_item.dart [hunk 6]
```diff
@@ -60,10 +59,17 @@ class CatalogItem extends StatelessWidget {
...
-                        style: theme.transactionAmountStyle,
-                      )
+                      RichText(
+                        text: TextSpan(
+                          style: theme.transactionAmountStyle,
+                          children: <InlineSpan>[
+                            _buildCurrencySymbol(),
+                            TextSpan(
+                              text: _formattedPrice(), // Price
+                            ),
```
Comment (erdemyerebasmaz) [inline lib/routes/charge/items/catalog_item.dart]: You can't use different font family & size for parts of text in single Text widget.
To overcome this issue symbol and price is being rendered separately under RichText.

File: lib/routes/charge/items/catalog_item.dart [hunk 7]
```diff
@@ -83,12 +89,38 @@ class CatalogItem extends StatelessWidget {
...
+              ),
+            ),
+          )
+        : TextSpan(
+            text: _getSymbol(), // Icon
+          );
+  }
+
+  String _getSymbol() {
+    var currency = Currency.fromTickerSymbol(_itemInfo.currency);
```
Comment (roeierez) [inline lib/routes/charge/items/catalog_item.dart]: Instead you can use the `CurrencyWrapper.fromShortName` to get the unified API for different currencies.
I makes sense also to add `symbol` property on the `CurrrencyWrapper`.

File: lib/routes/charge/items/catalog_item.dart (hunk 5 above)
Comment (roeierez) [reply lib/routes/charge/items/catalog_item.dart]: It is better to fix the font. @yaslama  is working on this. In the meanwhile can you use "S" character for satoshi and remove the work around?

Comment (erdemyerebasmaz) [reply lib/routes/charge/items/catalog_item.dart]: [Done](https://github.com/breez/breezmobile/pull/292/commits/9120cc013e7616ee53f12e4c7b3abba168d95cc6)

File: lib/routes/charge/items/catalog_item.dart (hunk 7 above)
Comment (erdemyerebasmaz) [reply lib/routes/charge/items/catalog_item.dart]: [Done](https://github.com/breez/breezmobile/pull/292/commits/a5949de69578d9dcc7dac4b8ac73ce14e8784a80)

**Pull request merged**